src/components/AccountCard.vue

```diff
@@ -29,14 +29,16 @@ function focusInput(event: MouseEvent) {
 
 <template>
     <div class="account-card">
-        <a class="admin-switch" href="javascript:void(0)" @click="account.admin = !account.admin" title="Changer le rôle">
+        <a class="admin-switch" href="javascript:void(0)" @click="dataStore.editAccount(id, 'admin', !account.admin)"
+            title="Changer le rôle">
             <img v-if="account.admin" src="../assets/fireman-helmet.png">
             <img src="../assets/svg/user.svg">
         </a>
-        <input name="mail" @input="" :value="account.mail" />
+        <input class="input-style" type="email" name="mail"
+            @input="dataStore.editAccount(id, 'mail', ($event.target as HTMLInputElement).value)" :value="account.mail" />
         <div>
             <a href="javascript:void(0)" @click="focusInput"><img src="../assets/svg/edit.svg"></a>
-            <a href="javascript:void(0)" @click=""><img src="../assets/svg/delete.svg"></a>
+            <a href="javascript:void(0)" @click="dataStore.deleteAccount(id)"><img src="../assets/svg/delete.svg"></a>
```

Comment:
For clarity create function in <script> invoked in template, and then call store in them